### ISSUE TITLE
Support ruby-3.0.0

### ISSUE THREAD

**Issue by jiikko** [DeployGate/deploygate-cli#292]
It was installed 0.6.1 if ruby-3.0.0.

```shell
$ ruby -v
ruby 3.0.0p0 (2020-12-25 revision 95aff21468) [x86_64-darwin20]

$ gem i deploygate
Fetching deploygate-0.6.1.gem

dg installed! To get started fast:

  $ dg deploy

Successfully installed deploygate-0.6.1
Parsing documentation for deploygate-0.6.1
Installing ri documentation for deploygate-0.6.1
Done installing documentation for deploygate after 0 seconds
1 gem installed
```

```
Bundler found conflicting requirements for the Ruby version:
  In Gemfile:
    Ruby x86_64-darwin-20

    bundler (>= 2.1.4, < 3.0) x86_64-darwin-20 was resolved to 2.2.3, which depends on
      Ruby (>= 2.3.0) x86_64-darwin-20

    deploygate x86_64-darwin-20 was resolved to 0.8.3, which depends on
      fastlane (~> 2.148.1) was resolved to 2.148.1, which depends on
        Ruby (>= 2.4)

    deploygate x86_64-darwin-20 was resolved to 0.8.3, which depends on
      fastlane (~> 2.148.1) was resolved to 2.148.1, which depends on
        google-api-client (< 0.39.0, >= 0.37.0) was resolved to 0.38.0, which depends on
          Ruby (~> 2.4)
```

The workaround is to use ruby-2.x.

**Comment by henteko** [comment]
dg command depends on fastlane.
We wait for fastlane ruby 3.0.0 support to be completed.
https://github.com/fastlane/fastlane/issues/17931

**Comment by ainame** [comment]
👋 We released v2.183.0 which unblocks Ruby 3 installation. Hope it works for this project🤞 
https://github.com/fastlane/fastlane/releases/tag/2.183.0

**Comment by jmatsu** [comment]
Closed by #321
